scripts/languageserver/packages/AbstractTrees

```diff
@@ -1 +0,0 @@
-Subproject commit 74695d4634c2286e20c75accb2739d07344fe56a
```

Comment:
Something is wrong here with the submodules, this shouldn't reference local paths from your machine... Also, when I checkout this branch all the submodules don't update correctly, which I assume is due to whatever is going on here...